R-packages/evalcast/R/get_covidhub_predictions.R

```diff
@@ -151,11 +151,7 @@ get_forecast_dates <- function(forecasters,
   forecast_dates <- as_date(forecast_dates)
   forecaster_dates <- vector("list", length = length(forecasters))
   for (i in seq_len(length(forecasters))) {
-    forecaster_dates[[i]] <- tryCatch({
-      lubridate::as_date(get_covidhub_forecast_dates(forecasters[i]))
-    },
-    error = function(e) cat(sprintf("%i. %s\n", i, e$message))
-    )
+    forecaster_dates[[i]] <- lubridate::as_date(get_covidhub_forecast_dates(forecasters[i]))
```

Comment:
mostly out of scope, but if you know off the top of your head, im curious: why are we using lubridate here instead of just as_date?

Reply:
I'm not sure. (All the other `as_date`s should really be loading directly from `lubridate` since we're not importing the function.)